java/java-tests/testData/codeInsight/daemonCodeAnalyzer/quickFix/optionalIsPresent/afterAnnotation.java

```diff
@@ -1,4 +1,4 @@
-// "Replace Optional.isPresent() condition with functional style expression" "true-preview"
```

Comment:
I see unrelated changes. E. g., removal of `-preview` here. Please rebase your patch on top of master properly to avoid unrelated changes. Thanks.

Reply:
Fixed